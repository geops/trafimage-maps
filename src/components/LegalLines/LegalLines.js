import React from "react";
import PropTypes from "prop-types";
import Contact from "./Contact";
import Legal from "./Legal";
import Imprint from "./Imprint";

const kontakt = Contact;
const rechtliches = Legal;
const impressum = Imprint;

const docs = {
  kontakt,
  rechtliches,
  impressum,
};

const propTypes = {
  language: PropTypes.string,
  doc: PropTypes.oneOf(Object.keys(docs)),
};

const defaultProps = {
  doc: Object.keys(docs)[0],
  language: "de",
};

function LegalLines({ language, doc }) {
  const Comp = docs[doc];
  return <Comp language={language} />;
}

LegalLines.propTypes = propTypes;
LegalLines.defaultProps = defaultProps;

export default LegalLines;
