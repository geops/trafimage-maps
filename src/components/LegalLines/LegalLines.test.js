import React from "react";
import { render } from "@testing-library/react";
import LegalLines from "./LegalLines";

describe("LegalLines", () => {
  test("uses default  properties", () => {
    window.console.error = jest.fn().mockImplementation(() => {});
    const component = render(<LegalLines />);
    expect(component.container.innerHTML).toMatchSnapshot();
  });

  ["rechtliches", "impressum", "kontakt"].forEach((doc) => {
    ["de", "fr", "en", "it"].forEach((lng) => {
      test(`should match snapshot with doc=${doc} and language=${lng}`, () => {
        const component = render(<LegalLines doc={doc} language={lng} />);
        expect(component.container.innerHTML).toMatchSnapshot();
      });
    });
  });
});
