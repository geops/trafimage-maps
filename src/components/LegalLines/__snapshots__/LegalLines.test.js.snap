// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LegalLines should match snapshot with doc=impressum and language=de 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Gesamtverantwortung Trafimage.
    </b>
    <br>
    SBB AG
    <br>
    Infrastruktur – Anlagen und Technologie – Operation Center Technik – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Schweiz
    <br>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Betrieb, Wartung und Weiterentwicklung der Trafimage-Anwendungen, Datenintegration.
    </b>
    <br>
    geOps AG
    <br>
    Solothurnerstrasse 235
    <br>
    CH-4600 Olten
    <br>
    <a href="http://www.geops.ch">
      www.geops.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Karten- und Bahnhofplanerstellung.
    </b>
    <br>
    evoq communications AG
    <br>
    Ottikerstrasse 59
    <br>
    CH-8006 Zürich
    <br>
    <a href="http://www.evoq.ch">
      www.evoq.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Verantwortlich für die Inhalte der Webkarten:
    </b>
    Die Zuständigkeit und Kontaktangabe für die einzelnen Kartenthemen sind im Info-Pop-up
    <svg stroke="currentColor"
         fill="currentColor"
         stroke-width="0"
         viewbox="0 0 512 512"
         height="1em"
         width="1em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z">
      </path>
    </svg>
    in der entsprechenden Menüzeile aufgeführt.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=impressum and language=en 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Overall responsibility for Trafimage.
    </b>
    <br>
    SBB AG
    <br>
    Infrastructure – Installations and Technology – Technology Operation Center – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Switzerland
    <br>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Operation, maintenance and further development of the Trafimage applications, data integration.
    </b>
    <br>
    geOps AG
    <br>
    Solothurnerstrasse 235
    <br>
    CH-4600 Olten
    <br>
    <a href="http://www.geops.ch">
      www.geops.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Map and station plan production.
    </b>
    <br>
    evoq communications AG
    <br>
    Ottikerstrasse 59
    <br>
    CH-8006 Zürich
    <br>
    <a href="http://www.evoq.ch">
      www.evoq.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Responsibility for the content of the web maps:
    </b>
    The responsibility and contact details for the individual map topics are listed in the info pop-up
    <svg stroke="currentColor"
         fill="currentColor"
         stroke-width="0"
         viewbox="0 0 512 512"
         height="1em"
         width="1em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z">
      </path>
    </svg>
    in the corresponding menu bar.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=impressum and language=fr 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Ensemble de la responsabilité Trafimage.
    </b>
    <br>
    CFF SA
    <br>
    Infrastructure – Installations et technologie – Operation Center Technique – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Suisse
    <br>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Exploitation, maintenance et développement des applications Trafimage, intégration des données.
    </b>
    <br>
    geOps AG
    <br>
    Solothurnerstrasse 235
    <br>
    CH-4600 Olten
    <br>
    <a href="http://www.geops.ch">
      www.geops.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Établissement des cartes et des plans des gares.
    </b>
    <br>
    evoq communications AG
    <br>
    Ottikerstrasse 59
    <br>
    CH-8006 Zürich
    <br>
    <a href="http://www.evoq.ch">
      www.evoq.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Responsables des contenus des cartes Web:
    </b>
    La compétence et les coordonnées de contact pour chaque thème de la carte sont indiquées dans le pop-up d'information
    <svg stroke="currentColor"
         fill="currentColor"
         stroke-width="0"
         viewbox="0 0 512 512"
         height="1em"
         width="1em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z">
      </path>
    </svg>
    dans la ligne de menu correspondante.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=impressum and language=it 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Responsabilità generale Trafimage.
    </b>
    <br>
    FFS SA
    <br>
    Infrastruttura – Impianti e tecnologia – Operation Center Tecnica – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Svizzera
    <br>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Gestione, manutenzione e ulteriore sviluppo delle applicazioni Trafimage, integrazione dei dati.
    </b>
    <br>
    geOps AG
    <br>
    Solothurnerstrasse 235
    <br>
    CH-4600 Olten
    <br>
    <a href="http://www.geops.ch">
      www.geops.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Realizzazione di cartine e piante delle stazioni.
    </b>
    <br>
    evoq communications AG
    <br>
    Ottikerstrasse 59
    <br>
    CH-8006 Zürich
    <br>
    <a href="http://www.evoq.ch">
      www.evoq.ch
    </a>
    <br>
  </p>
  <p>
    <b>
      Responsabilità per i contenuti delle cartine:
    </b>
    La responsabilità e i dettagli di contatto per i singoli argomenti della mappa sono elencati nel pop-up delle informazioni
    <svg stroke="currentColor"
         fill="currentColor"
         stroke-width="0"
         viewbox="0 0 512 512"
         height="1em"
         width="1em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z">
      </path>
    </svg>
    nella barra dei menu corrispondente.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=kontakt and language=de 1`] = `
<div class="tm-imprint">
  <p>
    Haben Sie Fragen, Verbesserungsvorschläge, Hinweise auf Unstimmigkeiten oder Interesse an Karten und Plänen? Dann nehmen Sie bitte mit uns Kontakt auf!
  </p>
  <p>
    <b>
      SBB AG
    </b>
    <br>
    Infrastruktur – Anlagen und Technologie – Operation Center Technik – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Schweiz
  </p>
  <p>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
    <a href="mailto:trafimage@sbb.ch">
      trafimage@sbb.ch
    </a>
    <br>
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=kontakt and language=en 1`] = `
<div class="tm-imprint">
  <p>
    Do you have any questions about our maps and plans or suggestions for improvement? Or would you like to point out any errors? If so, please contact us.
  </p>
  <p>
    <b>
      SBB AG
    </b>
    <br>
    Infrastructure – Installations and Technology – Technology Operation Center – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Switzerland
  </p>
  <p>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
    <a href="mailto:trafimage@sbb.ch">
      trafimage@sbb.ch
    </a>
    <br>
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=kontakt and language=fr 1`] = `
<div class="tm-imprint">
  <p>
    Vous avez des questions, des suggestions, des remarques sur des inexactitudes ? Les cartes et plans vous intéressent? N’hésitez pas à nos contacter.
  </p>
  <p>
    <b>
      CFF SA
    </b>
    <br>
    Infrastructure – Installations et technologie – Operation Center Technique – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Suisse
  </p>
  <p>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
    <a href="mailto:trafimage@cff.ch">
      trafimage@cff.ch
    </a>
    <br>
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=kontakt and language=it 1`] = `
<div class="tm-imprint">
  <p>
    Avete domande, proposte di miglioramento, segnalazioni di inesattezze o siete interessati.
  </p>
  <p>
    <b>
      FFS SA
    </b>
    <br>
    Infrastruttura – Impianti e tecnologia – Operation Center Tecnica – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Svizzera
  </p>
  <p>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
    <a href="mailto:trafimage@ffs.ch">
      trafimage@ffs.ch
    </a>
    <br>
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=rechtliches and language=de 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Rechtlicher Hinweis und Datenschutz.
    </b>
  </p>
  <p>
  </p>
  <p>
    Es gelten die
    <a href="http://www.sbb.ch/meta/rechtlicher-hinweis.html">
      rechtlichen Hinweise
    </a>
    und
    <a href="http://www.sbb.ch/meta/datenschutz.html">
      Datenschutzbestimmungen
    </a>
    der SBB.
  </p>
  <p>
    In Abweichung zu den allgemeinen Datenschutzbestimmungen der SBB werden auf maps.trafimage.ch unpersönliche Nutzungsdaten wie Ihre IP-Adresse, die zuletzt besuchte Seite, der verwendete Browser, Datum, Uhrzeit etc. gespeichert und mit
    <a href="http://matomo.org/">
      Matomo
    </a>
    und
    <a href="https://www.piano.io/">
      Piano
    </a>
    anonymisiert ausgewertet.
  </p>
  Bei Kartenthemen und –Layer in der internen Sicht handelt es sich um Informationen für den internen Gebrauch gemäss "R K 400.16 Informationsklassifizierung." und "R K 040.1 Konzernweisung Datenschutz.".
  <p>
  </p>
  <p>
    <b>
      Erscheinungsbild von Trafimage.
    </b>
  </p>
  <p>
  </p>
  <p>
    Das Erscheinungsbild von Trafimage (Bildaufbau und Bildgestaltung) ist ein fester Bestandteil des Auftritts der SBB und ist urheberrechtlich geschützt. Die auf den Websites von Trafimage enthaltenen Karten und Pläne inklusive den dazugehörenden Informationen werden ausschliesslich für den privaten Gebrauch sowie zu Informationszwecken bereit gestellt.
  </p>
  <p>
    <b>
      Nutzung Trafimage-Kartenbild und Kartendaten von swisstopo und OpenStreetMap.
    </b>
  </p>
  <p>
  </p>
  <p>
    Für die Geobasisdaten (z.B. Luftbilder, Relief, Landeskarten) gelten die Bestimmungen von swisstopo und OpenStreetMap. Es ist gestattet, einzelne Karten und Pläne bzw. Ausschnitte davon und Informationen für private Zwecke herunterzuladen, abzuspeichern, auszudrucken oder auf andere Weise zu vervielfältigen sofern weder der Copyrightvermerke "© SBB/CFF/FSS, Geodaten: © OpenStreetMap contributors, © swisstopo" noch andere gesetzlich geschützte Bezeichnungen entfernt werden. Sämtliche Eigentums- bzw. Nutzungsrechte verbleiben bei der SBB (Kartenbild) und bei der swisstopo (Kartengrundlagen). Sie können diese Zustimmung jederzeit widerrufen. Jegliche weitere Nutzung (vollständig oder teilweise), Übermittlung (elektronisch oder mit anderen Mitteln), Modifikation, Verknüpfung oder Benutzung der in den Websites enthaltenen Geobasisdaten ist nur mit vorheriger schriftlicher Einwilligung von SBB und swisstopo gestattet.
  </p>
  <p>
    <b>
      Copyright swisstopo.
    </b>
  </p>
  <p>
  </p>
  <p>
    Die in dieser Internet-Plattform integrierten Geobasisdaten sind Eigentum der Schweizerischen Eidgenossenschaft. Die Daten sind urheberrechtlich durch das Bundesgesetz über Geoinformation (SR 510.62) geschützt. Sofern nichts anderes vermerkt ist, handelt es sich um Daten des Bundesamtes für Landestopografie (swisstopo) und es gelten die Bestimmungen der Geoinformationsverordnung (SR 510.620) und die Verordnung des VBS über die Gebühren des Bundesamtes für Landestopografie (SR 510.620.2).
  </p>
  <p>
    <b>
      Haftung von swisstopo.
    </b>
  </p>
  <p>
  </p>
  <p>
    Liegt ein Vertragsverhältnis zwischen swisstopo und dem Nutzer eines Dienstes vor, so haftet swisstopo nur für Schäden, die sie grobfahrlässig oder vorsätzlich verursacht hat und im Verantwortungsbereich von swisstopo liegen. swisstopo haftet nicht für entgangenen Gewinn, Datenverluste oder sonstige indirekte Schäden, die namentlich durch Fehlinformationen oder technische Störungen entstanden sind. swisstopo übernimmt keine Verantwortung für Fehler, die nicht in Ihrem Verantwortungsbereich liegen.
  </p>
  <p>
    <b>
      Daten der SBB und ihrer Partner.
    </b>
  </p>
  <p>
  </p>
  <p>
    Alle anderen Daten sind Eigentum der SBB oder ihrer Partner.
  </p>
  <p>
    <b>
      Zeichnung.
    </b>
  </p>
  <p>
    Keine Garantie der Verfügbarkeit der abgespeicherten Zeichnungen, kein Backup, Speicherung nur für 1 Jahr, keine Haftung seitens Betreiber SBB für abgespeicherte Daten; Zeichnung kann jederzeit vom Erstellenden gelöscht werden; Dritte können auf Zeichnung zugreifen (betrachten und kopieren, jedoch nicht verändern); Der Nutzer verpflichtet sich, die zur Verfügung stehende Zeichnungsfunktion nicht missbräuchlich zu verwenden und rechtswidrige Handlungen zu unterlassen. Datenschutz- und urheberrechtliche Vorschriften sind zu beachten; ein Missbrauch wird geahndet und kann strafrechtlich verfolgt werden. Insbesondere darf Schutz von Privatsphäre / persönlichen Daten nicht verletzt und Urheberrechte nicht verletzt werden. Die SBB behält sich vor, solche Daten jederzeit ohne Warnung zu löschen.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=rechtliches and language=en 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Legal information and data protection.
    </b>
  </p>
  <p>
  </p>
  <p>
    SBB's
    <a href="http://www.sbb.ch/en/meta/legal-information.html">
      legal information
    </a>
    and
    <a href="http://www.sbb.ch/en/meta/data-protection.html">
      Privacy Policy
    </a>
    apply.
  </p>
  <p>
    Contrary to SBB’s general data protection provisions, maps.trafimage.ch stores non-personal usage data such as your IP address, the page you last visited, the browser used, the date, the time, etc. and analyses it in an anonymised form using
    <a href="http://matomo.org/">
      Matomo
    </a>
    and
    <a href="https://www.piano.io/">
      Piano
    </a>
    .
  </p>
  <p>
  </p>
  <p>
    <b>
      Corporate identity of Trafimage.
    </b>
  </p>
  <p>
  </p>
  <p>
    The corporate identity of Trafimage (structure and design) forms an integral part of SBB's image and is protected by copyright. The maps and plans contained on the websites of Trafimage, including the related information, are supplied solely for private use and for information purposes.
  </p>
  <p>
    <b>
      Use of Trafimage map images and swisstopo and OpenStreetMap map data.
    </b>
  </p>
  <p>
  </p>
  <p>
    The provisions of swisstopo and OpenStreetMap and apply to basic geographical data (e.g. aerial photographs, relief data, national maps). Individual maps and plans, or excerpts thereof, and information may be downloaded, saved, printed or copied by other methods for private use provided that none of the copyright symbols "© SBB / Geodaten: © OpenStreetMap contributors, © swisstopo" or other legally protected names are removed. All property rights and rights of use respectively shall remain with SBB (map images) and swisstopo (basic map data), which can revoke their consent at any time. None of the basic geographical data contained on the websites may be reproduced (completely or in part), transmitted (electronically or by other means), modified, linked or used without the prior written consent of SBB and swisstopo.
  </p>
  <p>
    <b>
      Copyright of swisstopo.
    </b>
  </p>
  <p>
  </p>
  <p>
    The basic geographical data integrated into this Internet platform is the property of the Swiss Confederation. The data is copyright protected by the Swiss Federal Law on Geoinformation (SR 510.62). Unless otherwise indicated, data is provided by the Federal Office of Topography (swisstopo) and is subject to the provisions of the Geoinformation Ordinance (SR 510.620) and the ordinance of the Federal Department of Defence, Civil Protection and Sport on fees and charges of the Federal Office of Topography (SR 510.620.2).
  </p>
  <p>
    <b>
      Liability on the part of swisstopo.
    </b>
  </p>
  <p>
  </p>
  <p>
    If a contractual relationship between swisstopo and the user of a service exists, swisstopo shall only be liable for damage it has caused by gross negligence or deliberate intent which lies within the scope of its responsibility. swisstopo shall not be liable for loss of earnings, data loss or other indirect damage due in particular to incorrect information or technical malfunctions. swisstopo assumes no responsibility for errors that do not lie within its area of responsibility.
  </p>
  <p>
    <b>
      Data belonging to SBB and its partners.
    </b>
  </p>
  <p>
  </p>
  <p>
    All other data is the property of SBB or its partners.
  </p>
  <p>
    <b>
      Draw.
    </b>
  </p>
  <p>
    No guarantee for availability of stored drawings, no backup provided, storage for 1 year only, no liability of operator SBB/CFF/FFS for stored data. Drawing can be deleted anytime by original creator; Third parties/public can access drawing (view and copy, but not change). The user commits himself not to misuse the provided drawing function and to refrain from illegal activities. Privacy and copyright regulations must be respected; misuse is punishable and can be prosecuted. In particular, protection of privacy , personal data and copyrights must not be violated. SBB reserves the right to delete such data at any time without warning.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=rechtliches and language=fr 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Mention juridique et protection des données.
    </b>
  </p>
  <p>
    La
    <a href="http://www.cff.ch/meta/mention-juridique.html">
      Mention juridique
    </a>
    et les
    <a href="http://www.cff.ch/meta/protection-des-donnees.html">
      dispositions sur la protection des données
    </a>
    des CFF s’appliquent.
  </p>
  <p>
    En dérogation aux dispositions générales des CFF en matière de protection des données, maps.trafimage.ch enregistre des données d’utilisation non personnelles telles que votre adresse IP, la dernière page visitée, le logiciel de navigation utilisé, la date, l’heure, etc. et les exploite sous forme anonymisée avec
    <a href="http://matomo.org/">
      Matomo
    </a>
    et
    <a href="https://www.piano.io/">
      Piano
    </a>
    .
  </p>
  <p>
  </p>
  <p>
    <b>
      Image de Trafimage.
    </b>
  </p>
  <p>
    L’identité visuelle de Trafimage (constitution et composition de l’image) fait partie intégrante de la présentation des CFF et est protégée par un droit d’auteur. Les cartes et les plans contenus dans les sites Internet de Trafimage, y compris les informations y relatives, sont mis à disposition exclusivement à des fins d’usage privé et d’information.
  </p>
  <p>
    <b>
      Utilisation des cartes Trafimage et des données topographiques de swisstopo et OpenStreetMap.
    </b>
  </p>
  <p>
    Pour les géodonnées de base (vues aériennes, relief, cartes nationales), ce sont les dispositions de swisstopo et OpenStreetMap qui s’appliquent. Il est permis de télécharger, d’enregistrer, d‘imprimer, ou de reproduire sous toute autre forme, certaines cartes et plans et/ou des extraits ainsi que des informations, à condition de ne supprimer ni des mentions de copyright «© CFF/géodonnées: © OpenStreetMap contributors, © swisstopo» ni d’autres désignations légalement protégées. Les CFF conservent tous les droits de propriété ou bien d’utilisation (carte-image), de même que swisstopo (bases des cartes). Cette autorisation peut être révoquée à tout moment. Toute autre utilisation (en totalité ou en partie), transmission (par voie électronique ou avec d’autres moyens), modification, mise en lien ou utilisation des géodonnées de base nécessite l’autorisation préalable expresse des CFF et de swisstopo.
  </p>
  <p>
    <b>
      Copyright swisstopo.
    </b>
  </p>
  <p>
    Les géodonnées de base intégrées à cette plate-forme Internet sont la propriété de la Confédération suisse et protégées par un droit d’auteur, conformément à la loi fédérale sur la géoinformation (RS 510.62). Sauf indication contraire, il s’agit de données de l’Office fédéral de topographie (swisstopo) et les dispositions de l’Ordonnance sur la géoinformation (RS 510.620) ainsi que l’Ordonnance du DDPS sur les émoluments de l’Office fédéral de topographie (RS 510.620.2) s’appliquent.
  </p>
  <p>
    <b>
      Responsabilité swisstopo.
    </b>
  </p>
  <p>
    Si un contrat a été signé entre swisstopo et l’utilisateur d’un service, la responsabilité de swisstopo n’est engagée que pour des dommages liés à sa négligence ou causés intentionnellement et relevant de son domaine de compétence. Swisstopo ne peut être tenu pour responsable de toute perte de gain, toute perte de données ou tout autre dommage indirect, imputables notamment à une erreur d’information ou une panne technique. swisstopo décline toute responsabilité pour toute erreur ne relevant pas de son domaine de compétence.
  </p>
  <p>
    <b>
      Données des CFF et de leurs partenaires.
    </b>
  </p>
  <p>
    Toutes les autres données sont la propriété des CFF ou de leurs partenaires.
  </p>
  <p>
    <b>
      Dessin.
    </b>
  </p>
  <p>
    Aucune garantie n’est donnée quant à la disponibilité des dessins enregistrés, aucun backup n’est réalisé, le stockage est fonctionnel une année, aucune responsabilité n’est prise par l’exploitant SBB/CFF/FFS. Le dessin peut être en tout temps supprimé par son créateur. Des tiers peuvent accéder au dessin (visualiser et copier, mais pas modifier). L’utilisateur s’engage à ne pas abuser des fonctions de dessin disponibles et à renoncer à des activités illégales. La confidentialité et les droit d'auteur doivent être respectés; l’utilisation abusive est punissable et peut être poursuivie. En particulier, la protection de la vie privée / des données personnelles doit être conforme à la législation en vigueur. SBB/CFF/FFS se réserve le droit de supprimer un dessin à tout moment et sans avertissement.
  </p>
</div>
`;

exports[`LegalLines should match snapshot with doc=rechtliches and language=it 1`] = `
<div class="tm-imprint">
  <p>
    <b>
      Avvertenza legale e protezione dei dati.
    </b>
  </p>
  <p>
    Valgono le
    <a href="http://www.ffs.ch/meta/note-legali.html">
      note legali
    </a>
    e
    <a href="http://www.ffs.ch/meta/tutela-dei-dati.html">
      la dichiarazione sulla protezione dei dati
    </a>
  </p>
  <p>
    In deroga alle disposizioni generali in materia di protezione dei dati delle FFS, su maps.trafimage.ch i dati di utilizzo non personali come gli indirizzi IP, l’ultima pagina visitata, il browser utilizzato, la data, l’ora, ecc. vengono salvati e analizzati in modo anonimo con
    <a href="http://matomo.org/">
      Matomo
    </a>
    e
    <a href="https://www.piano.io/">
      Piano
    </a>
    .
  </p>
  <p>
  </p>
  <p>
    <b>
      Brand Trafimage.
    </b>
  </p>
  <p>
    Il brand Trafimage (struttura e presentazione delle immagini) è parte integrante del brand FFS e pertanto è protetto dal diritto d'autore. Le cartine e i piani contenuti nel sito web di Trafimage, comprese le relative informazioni, sono disponibili esclusivamente per uso privato e per scopi informativi.
  </p>
  <p>
    <b>
      Utilizzo dell’immagine cartografica Trafimage e dei dati swisstopo ed OpenStreetMap.
    </b>
  </p>
  <p>
    Per i geodati di base (p. es. immagini aeree, rilievi, carte topografiche) valgono le norme di swisstopo ed OpenStreetMap. A titolo privato è permesso scaricare, salvare, stampare o riprodurre in altro modo singole cartine e singoli piani o parte di essi e le relative informazioni, a condizione che non siano rimosse le indicazioni circa la fonte «© FFS / Geodati: © OpenStreetMap contributors, © swisstopo» né altre indicazioni tutelate per legge. Tutti i diritti d'autore rispettivamente di utilizzo rimangono alle FFS (immagine cartografica) e a swisstopo (dati cartografici), che possono in qualsiasi momento revocare il proprio assenso all'utilizzo. Qualsiasi ulteriore utilizzo (completo o parziale), trasmissione (elettronica o con altri mezzi), modifica, collegamento o impiego dei geodati di base contenuti nelle pagine web sono consentiti solo previo consenso scritto da parte di FFS e swisstopo.
  </p>
  min-height: 16px;
  <p>
    <b>
      Copyright swisstopo.
    </b>
  </p>
  <p>
    I geodati di base inclusi in questa piattaforma Internet sono proprietà della Confederazione svizzera. In virtù della Legge federale sulla geoinformazione (RS 510.62) questi dati godono di tutela giuridica per quanto riguarda il diritto d’autore. Se non indicato altrimenti, si tratta di dati dell'Ufficio federale di topografia (swisstopo) e quindi disciplinati dalle norme dell'Ordinanza sulla geoinformazione (OGI, RS 510.620) e dall\\’Ordinanza del DDPS sugli emolumenti dell’Ufficio federale di topografia (RS 510.620.2). Responsabilità di swisstopo. In presenza di un rapporto contrattuale tra swisstopo e l'utente di un servizio, swisstopo è responsabile solo dei danni che ha causato negligentemente o intenzionalmente nell'ambito delle proprie competenze. swisstopo declina ogni responsabilità per danni da mancato guadagno, perdita di dati o altri danni indiretti sorti in seguito a informazioni errate o guasti tecnici. swisstopo declina altresì ogni responsabilità per errori che esulano dall'ambito delle sue competenze.
  </p>
  <p>
    <b>
      Dati delle FFS e dei suoi partner.
    </b>
  </p>
  <p>
    Tutti gli altri dati sono proprietà delle FFS o dei suoi partner.
  </p>
  <p>
    <b>
      Disegnares.
    </b>
  </p>
  <p>
    La disponibilità dei disegni salvati non è garantita, nessun backup è operato, la memorizzazione dei disegni è limitata ad un anno, alcuna responsabilità può essere imputata all’operatore del servizio SBB/CFF/FFS. Il disegno può essere eliminato in qualsiasi momento da chi lo ha creato. Il disegno è accessibile a terzi (visualizzazione e/o copia. Modifica non ammessa). L’utente si impegna a non abusare del servizio ed a rinunciare ad un suo utilizzo per pratiche illegali. La confidenzialità ed il diritto d’autore devono essere rispettati. L’utilizzazione abusiva è punibile e può essere perseguita. In particolare, la protezione della sfera privata / dei dati personali deve essere conforme alle norme vigenti. SBB/CFF/FFS si riserva il diritto di eliminare un disegno in qualsiasi momento e senza avvertimento.
  </p>
</div>
`;

exports[`LegalLines uses default  properties 1`] = `
<div class="tm-imprint">
  <p>
    Haben Sie Fragen, Verbesserungsvorschläge, Hinweise auf Unstimmigkeiten oder Interesse an Karten und Plänen? Dann nehmen Sie bitte mit uns Kontakt auf!
  </p>
  <p>
    <b>
      SBB AG
    </b>
    <br>
    Infrastruktur – Anlagen und Technologie – Operation Center Technik – Technical Management
    <br>
    Fachbus Trafimage
    <br>
    Poststrasse 6
    <br>
    3072 Ostermundingen
    <br>
    Schweiz
  </p>
  <p>
    <a href="http://www.trafimage.ch">
      www.trafimage.ch
    </a>
    <br>
    <a href="mailto:trafimage@sbb.ch">
      trafimage@sbb.ch
    </a>
    <br>
  </p>
</div>
`;
